ensime_shared/ensime.py

```diff
@@ -23,6 +23,9 @@
 else:
     from Queue import Queue
 
+# Dictionary mapping a shorthand key to a vim command string, for various
```

Comment:
Where it's used wasn't obvious (although that's mostly because it's really easy to confuse the `vim_command` and `vim.command` invocations). I'm happy to remove, though. :)